membrain.cabal

```diff
@@ -50,6 +50,7 @@ common common-options
                        -fhide-source-paths
                        -Wmissing-export-lists
                        -Wpartial-fields
+                       -Wmissing-deriving-strategies
```

Comment:
Could you please put it under the conditional, so it won't produce warning about unknown pragma for earlier GHCs. Like it's done here:
https://github.com/kowainik/tomland/blob/295c2b7580534303723206ba651557adff385cf4/tomland.cabal#L58-L60

Reply:
Thank you. Pushed a commit fixing this.